### ISSUE TITLE
broken build: grisu_format is undefined

### ISSUE THREAD

**Issue by WarShoe** [fmtlib/fmt#1352]
Hi,

I have tried the most recent version of fmt, and despite the fact that fmt was compiled and installed without any problems, I have a link error "undefined grisu_format" while compiling my project.

**Comment by WarShoe** [comment]
... and another, CUDA-related build error:

/usr/bin/ld: ../../libfmt.a(format.cc.o): in function `fmt::v6::system_error::init(int, fmt::v6::basic_string_view<char>, fmt::v6::format_args)':
format.cc:(.text+0x1913): undefined reference to `std::runtime_error::operator=(std::runtime_error&&)'

**Comment by vitaut** [comment]
`grisu_format` is defined here: https://github.com/fmtlib/fmt/blob/a1079e9fd6f01b6c33ad6da999d2538c0607bbb5/include/fmt/format-inl.h#L889. Could you provide a complete example that reproduces the error?

**Comment by vitaut** [comment]
Closing as I was not able to repro but feel free to reopen with more details.

**Comment by WarShoe** [comment]
It looks like there was a conflict with an old version of fmt/spdlog. The naive "git pull; make; sudo make install" approach did not work, so I reported. But I managed to build my project with a fresh build/install with the option SPDLOG_FMT_EXTERNAL=ON. Sorry for the spam.

Still have some build problems on another node, but maybe it is just a misconfguration of my own project. Under investigation...
